frontend/package.json

```
@@ -22,6 +22,7 @@
     "react": "19.1.0",
     "react-dom": "19.1.0",
     "react-router": "7.5.3",
+    "swr-openapi": "^5.3.0",
```

```suggestion
    "swr-openapi": "5.3.0",
```

pin dependency, this allows for minor versions to be updated (e.g. `5.4.0`)